dojo/tools/qualys/parser.py

```diff
@@ -3,7 +3,7 @@
 import logging
 import html2text
 from . import utfdictcsv
-from defusedxml import ElementTree as etree
+from lxml import etree
```

Comment:
we are moving out of `lxml` and replacing it by `defusedxml` to address some security concerns around XML well-known attack. Why do you want to revert this work?

Reply:
Because it is not working. Currently the parser fails with "TypeError: __init__() got an unexpected keyword argument 'resolve_entities'". If we have a working alternative I am open for the replacement.

Reply:
that keyword param is not needed with defused: https://pypi.org/project/defusedxml/#external-entity-expansion-remote

Reply:
made some changes, @alles-klar take a look.